.gitignore

```diff
@@ -1,6 +1,9 @@
 # Created by https://www.gitignore.io/api/flask
 # Edit at https://www.gitignore.io/?templates=flask
 
+# Config file
+config.py
```

Comment:
You should update the README with instructions to create a config.py file.

Slightly nicer would be to use environment variables instead - add new variables to .env.template / .env and then fetch them from a python file with `os.getenv('YOUR_VARIABLE')`. You would still need to update the README to tell the reader to set the values in .env after running the setup script.